packs/BP/entities/network_link.json

```diff
@@ -0,0 +1,43 @@
+{
+    "format_version": "1.20.80",
+    "minecraft:entity": {
+        "description": {
+            "identifier": "fluffyalien_energisticscore:network_link",
+            "is_spawnable": false,
+            "is_summonable": true,
+            "is_experimental": false
+        },
+        "component_groups": {
+            "fluffyalien_energisticscore:despawn": {
```

Comment:
Using a component group and event to despawn this entity is unnecessary. Use [Entity#remove](https://learn.microsoft.com/en-us/minecraft/creator/scriptapi/minecraft/server/entity?view=minecraft-bedrock-stable#remove) instead.

Reply:
Never knew that existed. Neat.